cardano-node/src/Cardano/Node/Configuration/Logging.hs

```diff
@@ -187,6 +193,26 @@ createLoggingLayer ver nodeConfig' p = do
                                                   (nodeBasicInfo nodeConfig p nodeStartTime)
            >>= loadPlugin switchBoard
 
+         -- Forward all the metrics/peers/errors to 'TraceForwarderBK' using 'mapBackends'.
+         let metricsLogger = "cardano.node.metrics"
+             peersLogger   = "cardano.node.metrics.peersFromNodeKernel"
+             errorsLoggers = [ "cardano.node.AcceptPolicy"
+                             , "cardano.node.ChainDB"
+                             , "cardano.node.DnsResolver"
+                             , "cardano.node.DnsSubscription"
+                             , "cardano.node.ErrorPolicy"
+                             , "cardano.node.Handshake"
+                             , "cardano.node.IpSubscription"
+                             , "cardano.node.LocalErrorPolicy"
+                             , "cardano.node.LocalHandshake"
+                             , "cardano.node.Mux"
```

Comment:
@denisshevchenko, are we not interested in all errors?If so -- then getting all traces for cardano.node with `Error`+ severity could be indeed the way to go?

I'm a bit uneasy about this magic declaration of a list by fiat..

Reply:
Yes, we're interested in all errors. And I took this list from here: https://github.com/input-output-hk/cardano-node/blob/master/cardano-node/src/Cardano/Tracing/Tracers.hs#L278

Reply:
Done.